.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Before submitting your extensions, please *fully* read our contributing guidelines and our custom extension tutorial. Trust me, it is worthwhile and contains important information such as acceptance criteria and licensing.
```

Comment:
The "custom extension tutorial" is more widely referred to as the "custom extension _documentation_". Also consider adding a link for this and CONTRIBUTING.md.